test/distributed/org/apache/cassandra/distributed/test/RepairErrorsTest.java

```diff
@@ -20,27 +20,37 @@
 
 import java.io.IOException;
 import java.util.Collection;
-import java.util.UUID;
-
-import org.assertj.core.api.Assertions;
-import org.junit.Test;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
 
 import net.bytebuddy.ByteBuddy;
 import net.bytebuddy.dynamic.loading.ClassLoadingStrategy;
 import net.bytebuddy.implementation.MethodDelegation;
+import net.bytebuddy.implementation.bind.annotation.SuperCall;
+import org.apache.cassandra.db.SystemKeyspace;
```

Comment:
Looks like import ordering could use some love in this file.

Reply:
Ouch. Yeah, will clean this up.